dotnet/Utilities/Util.cs

```diff
@@ -48,57 +48,57 @@ public static string ReplaceNonAlphaNumeric(this string self, string replace)
 
         #region Writing CSV Data C# 
 
-        public static string EscapeQuotes(this string self)
-        {
-            return self?.Replace("\"", "\"\"") ?? "";
-        }
-
-        public static string Surround(this string self, string before, string after)
-        {
-            return $"{before}{self}{after}";
-        }
-
-        public static string Quoted(this string self, string quotes = "\"")
-        {
-            return self.Surround(quotes, quotes);
-        }
-
-        public static string QuotedCSVFieldIfNecessary(this string self)
-        {
-            return (self == null) ? "" : self.Contains('"') || self.Contains(',') ? self.Quoted() : self;
-        }
-
-        public static string ToCsvField(this string self)
-        {
-            return self.EscapeQuotes().QuotedCSVFieldIfNecessary();
-        }
-
-        public static string ToCsvRow(this IEnumerable<string> self)
-        {
-            return String.Join(",", self.Select(ToCsvField));
-        }
-
-        public static IEnumerable<string> ToCsvRows(this DataTable self)
-        {
-            yield return self.Columns.OfType<object>().Select(c => c.ToString()).ToCsvRow();
-            foreach (var dr in self.Rows.OfType<DataRow>())
-                yield return dr.ItemArray.Select(i => ToCsvField(i.ToString())).ToCsvRow();
-        }
-
-        public static void ToCsvFile(this DataTable self, string path)
-        {
-            File.WriteAllLines(path, self.ToCsvRows());
-        }
-
-        public static void ToCsvFile<T>(this IEnumerable<T> self, string path)
-        {
-            self.PropertiesToDataTable().ToCsvFile(path);
-        }
-
-        public static void ToCsvFile<K, V>(this IDictionary<K, V> self, string path)
-        {
-            self.PropertiesToDataTable().ToCsvFile(path);
-        }
+        //public static string EscapeQuotes(this string self)
```

Comment:
What happened here? @FrozenKiwi @mavimaec  do either of you know?

Reply:
Seems like a change @FrozenKiwi made to compile to the Magic Leap 13 days ago.

Reply:
Yeah, I commented it out because (I think) DataTable wasn't available

This was one of those "ToDo's" that meant to come back to, but lost track of.  I think we might need to move this out of this project somewhere that isn't consumed by Unity

Reply:
I reinstated this code in the latest change. Should be good now.